dashboard/app/controllers/api/v1/teacher_feedbacks_controller.rb

```diff
@@ -42,7 +42,7 @@ def count
     headers['csrf-token'] = form_authenticity_token
 
     @all_feedbacks_count = TeacherFeedback.where(
-      student_id: params.require(:student_id)
+      student_id: current_user.id
```

Comment:
i think this change makes sense, but can you update the comment on this method to match?